### PR TITLE
Name other supported languages on docs landing page

### PR DESCRIPTION
**Pull request by fg-j**
## Summary

The docs page leads directly to Getting Started guides for 4 popular language ecosystems. It might, however, be hard for users to discover other languages of interest. Let's link to some other supported languages.

## Checklist

* [x] I have viewed, signed, and submitted the Contributor License Agreement.
* [ ] I have linked issue(s) that this PR should close using keywords or the Github UI (See [docs](https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue))
* [ ] I have added an integration test, if necessary.
* [ ] I have reviewed the [styleguide](https://github.com/paketo-buildpacks/community/blob/main/STYLEGUIDE.md) for guidance on my code quality.
* [x] I'm happy with the commit history on this PR (I have rebased/squashed as needed).

### CODE REVIEW

File: content/docs/_index.md [hunk 1]
```diff
@@ -10,7 +10,12 @@ toc: false
 
 This section gets you started with Paketo Buildpacks using Paketo [**Builders**][builders], the **pack** CLI, and **Docker**.
 
-Select your favorite language and learn how to get started with Paketo Buildpacks.
+**_Select your favorite language and learn how to get started with Paketo
+Buildpacks._**
+
+Don't see your favorite language below? See the How To guides for even
+more languages, like [Golang][howto/Golang], [Ruby][howto/Ruby], or
+[PHP][howto/PHP].)
```
Comment (ForestEckhardt) [inline content/docs/_index.md:18]: There is a missing `(` and I am not entirely sure where it would go.

Comment (fg-j) [reply content/docs/_index.md:18]: oop, just a straggler...

Comment (fg-j) [reply content/docs/_index.md:18]:
```suggestion
[PHP][howto/PHP].
```

**Review by ForestEckhardt — APPROVED** [review]
lgtm

**Pull request merged**